docs/tutorial/index.md

```diff
@@ -16,6 +16,13 @@ Charmed 5G - it will save you many more hours later on. Follow the
 tutorial steps in sequence; they take you on a learning journey through the
 product.
 
+To complete this tutorial, you will need a machine with the following
+requirements:
+
+- A recent `x86_64` CPU with the `AVX-512` feature.
```

Comment:
```suggestion
- A `x86_64` CPU with the `AVX-512` feature (Intel 8th gen or more recent)
```

Reply:
This is now a bit misleading unfortunately. On the consumer market, most 12th gen will have the feature, but not the later production ones. 13th gen does not have it. On AMD's side, the Zen4 architecture seems to support it.

On the server market, it is a lot more true, but I think a lot of people will run this tutorial on a laptop or desktop, probably in a VM.

I just tried to validate on my laptop, and it seems I do not have the feature, on a 12th gen. I might be wrong with the feature set required, I will try to narrow it a bit further.

Thanks for the comment.

Reply:
I clarified the requirements for both Intel and AMD.